Sources/SwiftFormat/Rules/OmitExplicitReturns.swift

```diff
@@ -145,7 +145,7 @@ public final class OmitExplicitReturns: SyntaxFormatRule {
     CodeBlockItemListSyntax([
       CodeBlockItemSyntax(
         leadingTrivia: returnStmt.leadingTrivia,
-        item: .expr(returnStmt.expression!),
+        item: .expr(returnStmt.expression!.detached.with(\.trailingTrivia, [])),
```

Comment:
Since `returnStmt.expression` and `returnStmt` had the same `trailingTrivia`, formatting caused `trailingTrivia` to appear twice when present.
To fix this, I detached the `trailingTrivia` of `returnStmt.expression` when rewrapping with `CodeBlockItemSyntax`, ensuring it does not get duplicated.